utils/constants/acps.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package constants
+
+import "github.com/ava-labs/avalanchego/utils/set"
+
+// CurrentACPs is the set of ACPs that are currently, at the time of release,
+// marked as implementable.
+//
+// See: https://github.com/avalanche-foundation/ACPs/tree/main#readme
+var CurrentACPs = set.Of[uint32](
```

Comment:
I think 25 should also be listed here (it should also be marked as implementable in the ACP repo)?

Reply:
Just pinged @joshua-kim.